### ISSUE TITLE
Documentation question: Order of messages

### ISSUE THREAD

**Issue by stdedos** [PyCQA/pylint#8321]
### Bug description

In https://github.com/PyCQA/pylint/pull/7162, "we decreed" that the Problem Code goes first, then the Correct Code

It seems that https://github.com/PyCQA/pylint/pull/8287 somehow reverted it? 😕 

https://pylint.readthedocs.io/en/stable/user_guide/messages/warning/while-used.html (2.16.2)
![image](https://user-images.githubusercontent.com/133706/220279593-2bf69e93-6607-49e9-9934-ad8071e02f3c.png)

https://pylint.readthedocs.io/en/latest/user_guide/messages/warning/while-used.html (2.17.0-dev0)
![image](https://user-images.githubusercontent.com/133706/220279624-6edc5bff-4925-46b2-849b-e118a6ac57c8.png)

### Command used

```shell
.
```

### Pylint output

```shell
.
```

### Expected behavior

.

### Pylint version

```shell
.
```

**Comment by stdedos** [comment]
Perhaps

https://github.com/PyCQA/pylint/blob/899a7c595bca2936e89a87e8757c453bca687a24/doc/exts/pylint_messages.py#L87

```py
return "\n".join((bad_code, good_code, pylintrc, details, related)) + "\n" 
```

**Comment by DanielNoord** [comment]
This is correct. Would you want to open a PR to fix it?
